modules/Bio/EnsEMBL/Compara/RunnableDB/OrthologQM/CalculateWGACoverage.pm

```diff
@@ -144,7 +144,12 @@ sub run {
 		if ( defined $exon_ranges ){
 			foreach my $aln_mlss ( keys %{ $this_aln_range } ){
 				foreach my $gdb_id ( sort {$a <=> $b} keys %{ $orth_ranges } ){
-					my $combined_coverage = $self->_combined_coverage( $orth_ranges->{$gdb_id}, $this_aln_range->{$aln_mlss}->{$gdb_id}, $exon_ranges->{$gdb_id} );
+					# Make sure that if this is a component gdb_id it refers back to the principal. The exon data is all on the components and not on the principal, but the ortholog info and aln_mlss is only on principal
+					my $gdb = $gdba->fetch_by_dbID($gdb_id);
+					my $components = $gdb->component_genome_dbs($gdb_id);
```

Comment:
```suggestion
```
You don't seem to be using this for anything.

Reply:
Yes, I meant to remove that, thanks